### PR TITLE
Remove documentation for upcoming /pki/tidy-status endpoint.

### PR DESCRIPTION
**Pull request by victorr**
The documentation will be released along with the feature in Vault 1.10.

### CODE REVIEW

**Comment by victorr** [comment]
The documentation was originally added in https://github.com/hashicorp/vault/commit/51d3b79442414e023d51439e569f10ec022e0f7e .

File: website/content/docs/internals/telemetry.mdx [hunk 1]
```diff
@@ -289,40 +289,30 @@ These metrics relate to [Vault Enterprise Replication](/docs/enterprise/replicat
...
-| `secrets.pki.tidy.revoked_cert_total_entries`                                                | Number of entries in the certificate store for revoked certificates to verify during the tidy operation                                                                    | entry       | gauge   |
-| `secrets.pki.tidy.start_time_epoch`                                                          | Start time (as seconds since Jan 1 1970) when the PKI tidy operation is active, 0 otherwise                                                                                | seconds     | gauge   |
-| `secrets.pki.tidy.success`                                                                   | Number of times the PKI tidy operation has completed succcessfully                                                                                                         | operations  | counter |
-| `vault.secret.kv.count` (cluster, namespace, mount_point)                                    | Number of entries in each key-value secret engine.                                                                                                                         | paths       | gauge   |
```
Comment (stevendpclark) [inline website/content/docs/internals/telemetry.mdx:324]: Are these last two really part of the tidy api as well?

Comment (stevendpclark) [reply website/content/docs/internals/telemetry.mdx:324]: oh nvm, mis-read the diff...

**Review by stevendpclark — APPROVED** [review]

**Review by mladlow — APPROVED** [review]
I looked at the diff from the original PR and everything looked good to me. I went through the rows on the table and it seemed like everything was there.

**Comment by mladlow** [comment]
If you make the changes in this PR to main, you'll need to re-add the docs changes back to main eventually so they show up in the 1.10 docs, which you almost certainly know, but just in case/as a reminder.

**Pull request merged**